src/string/lipogram.rs

```diff
@@ -0,0 +1,78 @@
+use std::collections::HashSet;
+
+//*
+// Fn that returns the letters that are missing from the input slice
+// and are present in the english alphabet
```

Comment:
[`sum_of_digits.rs`](https://github.com/TheAlgorithms/Rust/blob/d3670a21067b87fae5fd8d67d6a06a051972d728/src/math/sum_of_digits.rs) has very good documentation (regarding the formatting and doc tests).